subsys/bluetooth/mesh/lpn.c

```diff
@@ -958,6 +962,8 @@ int bt_mesh_lpn_friend_update(struct bt_mesh_net_rx *rx,
 
 		BT_INFO("Friendship established with 0x%04x", lpn->frnd);
 
+		bt_mesh_heartbeat_send();
+
```

Comment:
Looks like the same issue is here, with the `friend_response_received` being called below.